### DISCUSSION TITLE
SSD202D Overclocking

### DISCUSSION THREAD

**Discussion by fifteenhex** [linux-chenxing/linux-chenxing.org#14] [General]
Stable @ 1.8GHz

```
# mhz
count=807053 us50=22893 us250=112662 diff=89769 cpu_MHz=1798.066
```
Crashes during boot @ 1.9GHz

Doesn't boot at all @ 2.0GHz.

**Comment by ltrsunny** [comment]
How about the thermal status?

**Reply by fifteenhex** [reply]
I didn't check at the time. When I get a chance I'll take some readings.
